touchforms/backend/xformplayer.py

```diff
@@ -427,7 +427,6 @@ def _parse_question (self, event):
                     Constants.DATATYPE_GEOPOINT: 'geo',
 
                     # not supported yet
-                    Constants.DATATYPE_DATE_TIME: 'datetime',
```

Comment:
@gcapalbo i doubt this make much difference for SMS, since it's not supported but thought I'd give you a heads up that I removed this in case SMS does special processing for date time questions